crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -116,3 +151,22 @@ impl From<domain::Quote> for Quote {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn test_factor() {
+        let values = vec![
+            (1.0, 0.01),
+            (2.0, 0.01),
+            (0.0, 0.0),
+            (-1.0, 0.0),
+            (0.001, 0.001),
+        ];
+        for (from, result) in values {
+            assert_eq!(f64::from(FeeFactor::partner_fee_capped_from(from)), result);
+        }
+    }
+}
```

Comment:
fine to test it out locally but I am not sure we want to use unit tests in the `domain`. I think the current decision is to not use it unless really necessary.